File: src/main/java/com/app/Ki_Data/security/auth/RegisterRequest.java
package com.app.Ki_Data.security.auth;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class RegisterRequest {
    private String name;
    private String last_name;
    private String email;
    private String password;
    @Builder.Default
    private String role="USER";
}

File: src/main/java/com/app/Ki_Data/security/auth/RegisterRequest.kt
package com.app.Ki_Data.security.auth

import com.app.Ki_Data.security.user.Role
import com.fasterxml.jackson.annotation.JsonProperty

data class RegisterRequest(
    val name: String,
    @JsonProperty("last_name") val lastName: String,
    val email: String,
    val password: String,
    val role: Role = Role.USER
)